docs/getting-started/application-setup/06-run-the-app.md

```diff
@@ -0,0 +1,62 @@
+# Run the App
+
+## Run the App as a Service Member
```

Comment:
❌ 
We should have some information of how to access the `*local` hosts locally by having developers edit their `/etc/hosts/` file.

```suggestion
# Run the App

## Accessing Web Applications

To access the MilMove, Office, Prime API, and Admin local domains during local
development, you will modify your `/etc/hosts` file to contain the following: 

    127.0.0.1 milmovelocal
    127.0.0.1 officelocal
    127.0.0.1 adminlocal
    127.0.0.1 primelocal

## Run the App as a Service Member
```

Reply:
Great callout! My computer already had this setup when I started.